app/components/channel_drawer/channel_drawer.js

```diff
@@ -91,37 +85,26 @@ export default class ChannelDrawer extends PureComponent {
     handleDrawerClose = () => {
         this.resetDrawer();
 
-        if (this.closeLeftHandle) {
-            InteractionManager.clearInteractionHandle(this.closeLeftHandle);
-            this.closeLeftHandle = null;
-        }
-
         if (this.state.openDrawer) {
-            this.setState({openDrawer: false});
-        }
-    };
-
-    handleDrawerCloseStart = () => {
-        if (!this.closeLeftHandle) {
-            this.closeLeftHandle = InteractionManager.createInteractionHandle();
+            // The state doesn't get updated if you swipe to close
```

Comment:
what does this mean? how that is affecting the behavior? 

I think that is why we used the interaction manager

Reply:
It's to handle how the Drawer doesn't update this.state.openDrawer when you swipe to open/close the drawer. The code didn't change; I just added the comment since it was confusing me at first

Reply:
I think the InteractionManager changes were for the runAfterInteractions calls, but I'm pretty sure the Drawer code does that itself

Reply:
be aware of this https://github.com/mattermost/mattermost-mobile/blob/master/app/components/channel_drawer/channel_drawer.js#L343

Reply:
Oh, I didn't notice that. Why aren't we using their code for this?

Reply:
We weren't because we needed to keep track of this, but perhaps that will work just fine after your changes

Reply:
I've switched the Drawer to register with the InteractionManager itself